pkg/vif/testdata/router/main.go

```diff
@@ -8,20 +8,31 @@ import (
 	"os"
 	"strings"
 
+	"github.com/datawire/dlib/dlog"
+	"github.com/sirupsen/logrus"
 	"github.com/telepresenceio/telepresence/v2/pkg/client"
 	"github.com/telepresenceio/telepresence/v2/pkg/vif"
 )
 
 func main() {
 	cfg := client.GetDefaultConfig()
-	ctx, cancel := context.WithCancel(client.WithConfig(context.Background(), &cfg))
+	bCtx := client.WithConfig(context.Background(), &cfg)
+	logger := logrus.New()
+	logger.SetLevel(logrus.DebugLevel)
+	bCtx = dlog.WithLogger(bCtx, dlog.WrapLogrus(logger))
+	vif.InitLogger(bCtx)
+
+	ctx, cancel := context.WithCancel(client.WithConfig(bCtx, &cfg))
 	defer cancel()
-	vif.InitLogger(ctx)
 	dev, err := vif.NewTunnelingDevice(ctx, nil)
```

Comment:
```suggestion
	dev, err := vif.NewTunnelingDevice(ctx, func(context.Context, tunnel.ConnID) (tunnel.Stream, error) {
		return nil, errors.New("stream routing not enabled; refusing to forward"")
	})
```
This way, you don't need a specific test in `vif.dispatchToStream()`.